packages/x-charts-pro/src/internals/material/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { ChartsBaseSlots, ChartsIconSlots } from '@mui/x-charts/models';
```

Comment:
Side by side with this file,  create a `readonly.spec.tsx` to ensure this doesn't break in the future 😆 

```tsx
import * as React from 'react';
import { BarChartPro } from '@mui/x-charts-pro/BarChartPro';

const settings = {
  height: 200,
  series: [{ data: [60, -15, 66, 68, 87, 82, 83, 85, 92, 75, 76, 50, 91] }],
  xAxis: [{ position: 'top' }],
  yAxis: [{ data: [1, 2, 3] }],
  margin: { top: 10, bottom: 20 },
} as const;

export const Component = () => {
  return (
    <BarChartPro
      {...settings}
      showToolbar
      slotProps={{
        baseDivider: {
          material: {
            sx: { backgroundColor: 'primary.main', height: 2 },
          },
        },
      }}
    />
  );
};
```